ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -2058,3 +2063,97 @@ - (BOOL)isEqual:(id)object {
 }
 
 @end
+
+#pragma mark - ORKLocationAnswerFormat
+
+@implementation ORKLocationAnswerFormat
```

Comment:
There is a warning that `init` has not been implemented. To dismiss the warning, go to `ORKAnswerFormat_Internal.h` and add it in at line 68.